### ISSUE TITLE
10/28 딥러닝 6주차 과제 : val_accuracy 88%이상

### ISSUE THREAD

**Issue by shsha42** [shsha42/TIL#2]
import numpy as np
import matplotlib.pyplot as plt
import tensorflow as tf
from tensorflow import keras
from tensorflow.keras import Sequential
from tensorflow.keras.layers import *

(X_train, y_train), (X_test, y_test) = keras.datasets.cifar10.load_data()

plt.figure()
plt.imshow(X_train[1])
plt.colorbar()

X_train = X_train/255.0
X_test = X_test/255.0

model = Sequential()
model.add(Conv2D(64, activation = 'relu', kernel_size = (3,3)))
model.add(MaxPooling2D(pool_size = (2, 2)))
model.add(Conv2D(32, activation = 'relu', kernel_size = (3, 3)))
model.add(Flatten(input_shape = (32, 32, 3)))
model.add(Dense(80, activation = 'relu'))
model.add(Dense(10, activation = 'softmax'))

model.compile(optimizer = 'adam', loss = 'sparse_categorical_crossentropy', metrics = ['accuracy'])

history = model.fit(X_train, y_train, epochs=10, verbose=1, validation_split=0.3)

plt.plot(history.history['loss'])
plt.plot(history.history['val_loss'])
plt.title('loss')
plt.ylabel('loss')
plt.xlabel('epoch')
plt.legend(['loss', 'val_loss'], loc = 'lower right')
plt.show()

plt.figure()
plt.imshow(X_test[0])
y_pred = model.predict(X_test)
print("정답=", y_test[0])
print("예측값=", y_pred[0])
predicted_class = np.argmax(y_pred[0])
print("예측 클래스 인덱스=", predicted_class)

# val_accuracy: 61%

![Image](https://github.com/user-attachments/assets/7ed288c2-4901-479f-8383-61580bcaa6be)

**Comment by shsha42** [comment]
# 최종

![Image](https://github.com/user-attachments/assets/b59b3201-66c9-4b1a-9e57-adb012725e15)
